tools/deb_to_rb.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
```

Comment:
Would it make sense to add switch options to add the name, description, homepage, etc.?